app/views/monitoring.html

```diff
@@ -260,6 +262,108 @@
+                            </div>
+                          </div>
+                          <div class="list-view-pf-additional-info">
+                            <div class="list-view-pf-additional-info-item">
+                              <span class="pficon fa-fw pficon-image"></span>
+                              <!-- TODO: not sure what to do with this, gotta figure it out -->
+                              <image-names pod-template="set.spec.template" pods="podsByOwnerUID[set.metadata.uid]"></image-names>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                    <div ng-repeat-end ng-if="expanded.statefulSets[set.metadata.name]" class="list-group-expanded-section" ng-class="{'expanded': expanded.statefulSets[set.metadata.name]}">
+
+                      <!-- TODO: add logs once we have logs subresource for statefulSets
+                      <log-viewer
+                        ng-if="'statefulsets/log' | canI : 'get'"
+                        object="pod"
+                        context="projectContext"
+                        options="logOptions.statefulSets[set.metadata.name]"
+                        empty="logEmpty.statefulSets[set.metadata.name]"
+                        run="logCanRun.statefulSets[set.metadata.name]"
+                        fixed-height="250"
+                        full-log-url="(set | navigateResourceURL) + '?view=chromeless'">
+
+                        <span class="container-details">
+                          <label for="selectLogContainer">Container:</label>
+
+                          <span ng-if="set.spec.containers.length === 1">
+                            {{set.spec.containers[0].name}}
+                          </span>
+
+                          <ui-select ng-init="logOptions.statefulSets[set.metadata.name].container = set.spec.containers[0].name" ng-show="set.spec.containers.length > 1" ng-model="logOptions.statefulSets[set.metadata.name].container" input-id="selectLogContainer">
+                            <ui-select-match>{{$select.selected.name}}</ui-select-match>
+                            <ui-select-choices repeat="container.name as container in set.spec.containers">
+                              <div ng-bind-html="container.name | highlight : $select.search"></div>
+                            </ui-select-choices>
+                          </ui-select>
+                        </span>
+
+                      </log-viewer>
+                      -->
+
+                      <!-- Until the metrics directive pulls the metrics from the time range of the life of the pod, hide the metrics for old stuff -->
+                      <div class="mar-top-lg" ng-if="metricsAvailable">
+                        Logs are not available for stateful sets.
```

Comment:
this doesnt seem right, this message only shows up if metricsAvailable is true, you'll still end up with any empty section when metrics arent available